### ISSUE TITLE
NTR - metabolism: [GDP-mannose biosynthetic process from glucose]

### ISSUE THREAD

**Issue by rozaru** [geneontology/go-ontology#27683]
_Please provide as much information as you can:_ 

* **Suggested term label:** GDP-mannose biosynthetic process from glucose

* **ChEBI ID and label of the substance:**
_Please ensure that the ChEBI ID corresponds to the physiological form (pH7.3) of the compound. In the ChEBI entry, pH7.3 compounds have a 'UniProt synonym', as in the example for [mycophenolate](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI%3A62932)_
has_primary_input	CHEBI:4167	D-glucopyranose
has_primary_output	CHEBI:21168	GDP-mannose

* **Definition (free text)**
The chemical reactions and pathways resulting in the formation of GDP-mannose from glucose.

* **Reference, in format PMID:#######**
PMID: 21273394
PMID: 9287314

* **Gene product name and ID to be annotated to this term**
In fly:
Hex-A
Hex-C
Pgi
Mpi
Pmm2
Gmppa
Gmppb

* **Parent term(s)**
GDP-mannose biosynthetic process GO:0009298
* **Synonym(s) (please specify, EXACT, BROAD, NARROW or RELATED)**
_Other than the standard synonyms below_

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

* **Cross-references**
MetaCyc  PWY-5659 (start with glucose 6-phosphate)

* **Any other information**
Cells can make GDP-mannose from either mannose or glucose. Depending on the cells, glucose is the predominant source.
Below is a schematic of the coverage of the new term compared to the already existing terms (the gene are from the fly).

![image](https://github.com/geneontology/go-ontology/assets/22237719/819d29c1-4c20-4579-aebf-fb274d1e3f97)

Thank you

Rossana

@sjm41
@ukemi
@deustp01

----

* **FOR ONTOLOGY EDITOR**

* **Logical definition**
* *  **substance metabolic process:**
'metabolic process'
 and ('has primary input or output' some [substance]
* * **substance biosynthetic process:**
'biosynthetic process'
 and ('has primary output' some [substance].
* * **substance catabolic process:**
'catabolic process'
 and ('has primary output' some [substance] 

* **Standard synonyms**
* *  **substance metabolic process:**
* * * EXACT: substance metabolism
* * **substance biosynthetic process:** 
* * * EXACT: substance anabolism
* * * EXACT: substance biosynthesis
* * * EXACT: substance formation
* * * EXACT: substance synthesis

* * **substance catabolic process:**    
* * * EXACT: substance breakdown
* * * EXACT: substance catabolism
* * * EXACT: substance degradation

**Comment by rozaru** [comment]
Missed the following parent term(s):
mannose catabolic process GO:0019309

**Comment by pgaudet** [comment]
@rozaru 

Just to make sure I understand: 

Are you suggesting that the new term 'GDP-mannose biosynthetic process' is_a mannose catabolic process? 

ie you'd consider mannose to mannose-6-phosphate a mannose catabolic process?

**Comment by rozaru** [comment]
@pgaudet,

I should have mention that this comment is related to another ticket #27682 where I suggested to move "GDP-mannose biosynthetic process from mannose" under "mannose catabolic process". 
If the new term "GDP-mannose biosynthetic process from glucose" is created it would go at the same level as "GDP-mannose biosynthetic process from mannose" is in the ontology.

This is the current ontology for mannose metabolic process:

mannose metabolic process 
           |__GDP-mannose biosynthetic process from mannose
           |__mannose biosynthetic process
           |__mannose catabolic process
           |   |__glycolytic process from mannose through fructose-6-phosphate
           |   |__mannose to fructose-6-phosphate catabolic process

This is my proposition:

mannose metabolic process 
           |__mannose biosynthetic process
           |__mannose catabolic process
           |   |__glycolytic process from mannose through fructose-6-phosphate
           |   |__mannose to fructose-6-phosphate catabolic process
           |   |__GDP-mannose biosynthetic process from mannose
           |   |__GDP-mannose biosynthetic process from glucose (new term)

It is a bit difficult to decide what constitutes a catabolic or a biosynthetic process when the output and input belong to multiple pathways. 
For "GDP-mannose biosynthetic process from mannose":
From the point of view of GDP-mannose this term cover a biosynthetic process
From the point of view of mannose this term cover its conversion into GDP-mannose which could been seen as a catabolic process.

**Comment by rozaru** [comment]
@pgaudet 
I realised that the commend above:
_Missed the following parent term(s):
mannose catabolic process GO:0019309_
is a mistake, as the potential new term is not part of the mannose metabolic process. Sorry about this.

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0141199
+name: GDP-mannose biosynthetic process from glucose
+namespace: biological_process
+def: "The chemical reactions and pathways resulting in the formation of GDP-mannose from glucose." [PMID:21273394, PMID:9287314]
+xref: MetaCyc:PWY-5659
+intersection_of: GO:0009058 ! biosynthetic process
+intersection_of: has_primary_input CHEBI:4167 ! D-glucopyranose
+intersection_of: has_primary_output CHEBI:21168 ! GDP-mannose
+relationship: starts_with GO:0004347 ! glucose-6-phosphate isomerase activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27683" xsd:anyURI
+created_by: pg
+creation_date: 2024-04-26T08:43:27Z

**Comment by rozaru** [comment]
@pgaudet 
I just notice a small detail, but shouldn't
+intersection_of: has_primary_output CHEBI:21168 ! GDP-mannose
be 
+intersection_of: has_primary_output CHEBI:57527 GDP-α-D-mannose(2−) ?

**Comment by pgaudet** [comment]
Thanks @rozaru ! 

I fixed this, and for the parent & siblings.

**Comment by sjm41** [comment]
@rozaru @ukemi @deustp01 @pgaudet 

Following the NTR in this ticket, we end up with this:
```
GDP-mannose biosynthetic process (GO:0009298) [Reactome:R-HSA-446205]
    |__GDP-mannose biosynthetic process from fructose-6-phosphate (GO:0061729)
    |__GDP-mannose biosynthetic process from glucose (GO:0141199)  [MetaCyc:PWY-5659]
    |__GDP-mannose biosynthetic process from mannose (GO:0061728)
```

Looking again now, I think the NTR actioned here (GO:0141199) should be merged with the pre-existing term 'GO:0061729 GDP-mannose biosynthetic process from fructose-6-phosphate.

Here's my reasoning:
1. If you look at Rossana's nice image in the original comment, the 'from fructose-6-phosphate' term is just describing the last 3 steps of the new 5-step 'from glucose' term.
2. Fig 1 and the accompanying text of PMID:9287314 Rossana mentions (shown below) says that "Man-6-P can be formed in two ways; either directly from mannose using hexokinase, or derived from the pathway of Glc → Glc-6-P ↔ Fru-6-P ↔ Man-6-P".  That is, there is one route from mannose, or another route from glucose via F6P.
3. Fig 1 of PMID:16339137 (used as the source for the 'from fructose-6-phosphate' (GO:0061729) term) also shows that the fructose-6-phosphate originates from Glucose, and again shows the same two routes - also shown below.
4. There are only 2 EXP annotations to the 'from fructose-6-phosphate' term (to human PMM2 and MPI), and they were both added recently by Rossana!


That said, there are still a few issues:
- the new 'from glucose' term currently includes "relationship: starts_with GO:0004347 ! glucose-6-phosphate isomerase activity", which would indicate it actually starts with glucose 6-phosphate, so there's an inconsistency with the name/def/LD there.
- That said, MetaCyc:PWY-5659 (xref on the new term GO:0141199) is stated to start from glucose-6-phosphate rather than glucose... (MetaCyc image shown below)
- R-HSA-446205 (Synthesis of GDP-mannose) shows GDP-mannose being made from either mannose or fructose-6-phosphate (so it's appropriate they xref to the parent term "GDP-mannose biosynthetic process"), but it doesn't show where the fructose-6-phosphate has originated from - i.e. it doesn't show the Glc → Glc-6-P ↔ Fru-6-P steps mentioned in the refs above. (reactome image also shown below)



Maybe we can talk about this in the weeds call tomorrow afternoon?

![Screenshot 2024-09-18 at 13 03 21](https://github.com/user-attachments/assets/447e6e96-6697-42f8-ae2b-69062b719ad8)

![Screenshot 2024-09-18 at 13 03 47](https://github.com/user-attachments/assets/415a22a2-c526-4d6a-921b-996e2c1bbba5)

![Screenshot 2024-09-18 at 13 05 26](https://github.com/user-attachments/assets/fe83e394-ec30-4aed-bb80-86ed354a8890)

![Screenshot 2024-09-18 at 13 06 15](https://github.com/user-attachments/assets/4ccafd0c-ab7c-4249-b29a-198096b778ef)

**Comment by sjm41** [comment]
From pathways2go discussion today:

Decision: Merge the new term (GDP-mannose biosynthetic process from glucose GO:0141199) into the pre-existing term (GDP-mannose biosynthetic process from fructose-6-phosphate GO:0061729), keeping the name and def of the pre-existing term. (That is, revert to what we had before, but add the MetaCyc:PWY-5659 xref to GO:0061729).
@pgaudet Can you take care of that please?

Justification: Although papers describe the second GDP-Man synthesis as starting from glucose, the first two steps (catalysed by HK and PGI) are best seen as being part of glycolysis. The resulting F(6)P is at a branch point - it can either continue to be metabolised in glycolysis (via PFK) or can be diverted into GFP-Man synthesis (using PMI, aka MPI). Though not perfect, this is the best (simplest) way of modularising and representing the information within GO-BP terms and GO-CAMs. (And it also mirrors the way Reactome models the situation.)

Other consequences: The way we are thinking about this specific example will hopefully help us with subsequent NTRs, and for reviewing other existing metabolic BP GO terms that appear to overlap or are nested in some way.... (e.g. based on the above arguments, there's a case for obsoleting "GO:0061619 glycolytic process from mannose through fructose-6-phosphate" and just keeping the "GO:0061611 mannose to fructose-6-phosphate catabolic process" term in order to connect mannose to core glycolysis.)
We will assess and review additional cases as we encounter them and aim to be consistent and establish guidelines!
